README.md

```diff
@@ -1,2 +1,35 @@
 # pizza-cutter-masking
 mask making codes for pizza-cutter
+
+Examples
+--------
+```bash
+# make the foregound/gaia/des stars/hyperleda mask
+pcm-make-mask \
+    --hyperleda hyperleda_B16_18.fits.gz \
+    --foreground foreground-v2.fits \
+    --gaia gaia.fits \
+    --des-stars gold_2_0_r_lt_21_summary.fit \
```

Comment:
Do we have notes here or elsewhere on how these inputs were generated? That may be on me if I passed them to you?

Reply:
I have notes for how the foreground was generated (from a db table) and how the mdet is generated (from the metadetect outputs). I'll add a file with these notes to the repo.

The des stars `gold_2_0_r_lt_21_summary.fit` is a dump of "good" gold objects with just position, mags and mash. I think it was done by Eli considering the extension is .fit rather than .fits . @erykoff 

gaia.fits is probably a DES table dump of the entire gaia table from DESDM

The original hyperleda was a query from their web site that I converted from csv to fits

Reply:
Yes, I got that file from @erykoff , the original was here

/nfs/slac/kipac/fs1/g/des/erykoff/des/y6a1/fgcm/run_v3.3.1-psf/background_rerun_y6_wfd/gold_rendering/gold_2_0_r_lt_21_summary.fit

Eli can I please get the query you used to put in our notes?